### DISCUSSION TITLE
Serial-numbered upgrade to app store needed?

### DISCUSSION THREAD

**Discussion by jwiede** [rsayfutdinov/Pikka#41] [Q&A]
Hello, I've got a serial-number-based license for Pikka, and now realize I need to upgrade to the AppStore version to continue using Pikka (apparently the prior isn't available any longer, to put on new machine).  Is there any sort of discount on upgrading to app store version available?

Thanks!
John W.
jfw@kompositor.com (so you can look up prior license, but I also have serial # if needed)

**Comment by RBHannam** [comment]
I have the same problem. Grateful for your advice.
Thanks.
